core/src/lib.rs

```diff
@@ -38,10 +38,17 @@ pub use chrono::{
 };
 
 pub use edn::{
+    Cloned,
     FromMicros,
+    FromRc,
     Keyword,
     ToMicros,
     Utc,
+    ValueRc,
+};
+pub use edn::parse::{
+    query as parse_query,
```

Comment:
If we're eliminating the query parser, I think I'd rather see the consumers of query parsing import `edn::parse::query` directly. It's the most clear expression of what's happening. Otherwise `core` is just going to be a bottleneck crate.

Reply:
This was done specifically for `query-algebrizer`, which doesn't import `edn` directly.  I'd prefer to keep it.